### ISSUE TITLE
RE: jekyll/_cci2/configuration-reference.md. How to use multiple jobs?

### ISSUE THREAD

**Issue by jcollado** [circleci/circleci-docs#836]
The documentation currently makes clear that there can be multiple jobs and that the default one is `build`. However, it doesn't mention what can be done with multiple jobs and all the examples I've seen only provide the `build` job.

Hence, my questions are what can be done with multiple jobs and how to use them.

**Comment by smart-alek** [comment]
@jcollado Thanks for the feedback! We're still in the midst of generating docs for 2.0, and you aren't the first to ask about multiple jobs, either. 😬 

The short answer: you can (theoretically) do almost anything with multiple jobs. :)

The long answer: right now we just have a `build` job set up, but eventually you might add a job for each type of test you run, like `integration-tests` or `acceptance-tests`.

The idea is to not force every step in the process to happen in lockstep. So you might define a `compile` step and have that run in a non-blocking fashion with your tests or build.

TL; DR: we're in the middle of building out support for these other types of jobs, so stay tuned, it's coming and it's called workflows. :)
